koku/masu/database/sql/reporting_ocpstoragelineitem_daily.sql

```diff
@@ -52,7 +55,8 @@ CREATE TEMPORARY TABLE reporting_ocpstoragelineitem_daily_{{uuid | sqlsafe}} AS
         li.persistentvolume,
         li.storageclass,
         li.persistentvolume_labels,
-        li.persistentvolumeclaim_labels
+        li.persistentvolumeclaim_labels,
+        nlid.node_labels
```

Comment:
I'm thinking about this, as to whether there's a good way to do this without grouping by the node labels. Right now if a record from the line item table happens to match multiple rows in the node label table then we'll likely duplicate some usage data.

Reply:
Updates applied